containers/payments/subscription/SubscriptionSection.js

```diff
@@ -83,37 +77,66 @@ const SubscriptionSection = ({ permission }) => {
         );
     }
 
+    const {
+        UsedDomains,
+        MaxDomains,
+        UsedSpace,
+        MaxSpace,
+        UsedAddresses,
+        MaxAddresses,
+        UsedMembers,
+        MaxMembers,
+        MaxVPN
+    } = organization || {};
+
     const { mailPlan, vpnPlan } = formatPlans(Plans);
     const { Name: mailPlanName } = mailPlan || {};
     const { Name: vpnPlanName } = vpnPlan || {};
-    const canRemoveCoupon = CouponCode && ![BUNDLE, BLACK_FRIDAY.COUPON_CODE].includes(CouponCode);
-    const i18n = getCyclesi18n();
-
-    /**
-     * Open subscription modal with criteria
-     * @param {String} action remove-coupon, yearly, upgrade, update
-     */
-    const handleModal = (action = '') => () => {
-        const coupon = action === 'remove-coupon' ? '' : CouponCode ? CouponCode : undefined; // CouponCode can equals null
-        const cycle = action === 'yearly' ? YEARLY : Cycle;
-        const plansMap = action === 'upgrade' ? { plus: 1, vpnplus: 1 } : toPlanNames(subscription.Plans);
 
+    const handleModal = () => {
         createModal(
             <SubscriptionModal
                 subscription={subscription}
-                plansMap={plansMap}
-                coupon={coupon}
+                plansMap={toPlanNames(subscription.Plans)}
+                coupon={CouponCode || undefined} // CouponCode can equals null
```

Comment:
this is quite irrelevant, but just for the sake of grammar :sweat_smile: : `can equal null`